src/pages/home/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Home.css';
+import {Link} from "react-router-dom";
+import Header from "../../components/header/Header";
+import redditLogo from '../../assets/logo.png';
+import {ReactComponent as LoaderImage} from "../../assets/reddit-app-loader.svg";
+import numberWithDots from "../../helpers/DotNotation";
+import TruncateString from "../../helpers/TruncateText";
+import Footer from "../../components/footer/Footer";
+
+function Home() {
+    const [posts, getPosts] = useState(null);
+    const [error, toggleError] = useState(false);
+    const [loading, toggleLoading] = useState(true);
+
+    useEffect(() => {
+        async function fetchData() {
+            toggleError(false);
+            try {
+                toggleLoading(true);
+                const result = await axios.get('https://www.reddit.com/hot.json?limit=15');
+                console.log(result.data);
+                getPosts(result.data);
+                toggleLoading(false);
+
+            } catch (e) {
+                console.error(e);
+                toggleError(true);
+            }
+        }
+
+        fetchData();
+    }, [])
+
+    return (
+        <>
+            <Header
+                logo={redditLogo}
+                title="Reddit"
+            />
+            <main className="outer-container">
+                <div className="inner-container">
+                    <h1>Hottest posts <span>on reddit right now</span></h1>
+                    <div className='posts'>
+
+                    {loading && <>
+                        <LoaderImage width="50px" height="50px" />
+                    </>
+                    }
```

Comment:
Dit kun je versimpelen
```suggestion
                    {loading && <LoaderImage width="50px" height="50px" />}
```